documentstore/restfulapi.py

```diff
@@ -217,6 +216,15 @@ class JournalAOPSchema(colander.MappingSchema):
 class DocumentsBundleSchema(colander.MappingSchema):
     """Representa o schema de dados para registro de Documents Bundle."""
 
+    @colander.instantiate(missing=colander.drop)
+    class publication_months(colander.MappingSchema):
+        month = colander.SchemaNode(colander.Int(), missing=colander.drop)
+
+        @colander.instantiate(missing=colander.drop)
+        class interval(colander.TupleSchema):
+            start_month = colander.SchemaNode(colander.Int(), missing=colander.drop)
```

Comment:
@jamilatta acho que quando o `interval` for passado nós devemos obrigar os campos `start_month` e `end_month`, eles não devem ser opcionais.

Reply:
@joffilyfe, mesmo com o missing para fazer o `drop`, é executado como esperado e é necessário informar 2 elementos.